src/Process/3dTilesProcessing.js

```diff
@@ -328,7 +335,10 @@ export function process3dTilesNode(cullingTest = $3dTilesCulling, subdivisionTes
 
 
         if (isVisible) {
-            node.cleanableSince = undefined;
+            if (node.cleanableSince) {
+                layer._cleanableTiles.splice(layer._cleanableTiles.indexOf(node), 1);
```

Comment:
Why remove this one ?

Reply:
Because it's not cleanable anymore - and I want the layer._cleanableTiles` array to only cleanable elements.

Reply:
Ok !